### ISSUE TITLE
border-box: box-sizing and animated padding not correct

### ISSUE THREAD

**Issue by Bellfalasch** [Prinzhorn/skrollr#241]
I had this div which was given the style height 100%, border-box box-sizing, and then padding 300 in the stylesheet. This works as the padding is taken away from the height, so the total area is still 100% (as of normal behaviour is adding padding on top the height).

So, It looked good. And for some reason I moved the padding from the stylesheet, and instead added it to my data-tag for animation. Start 300, end 150px. This however boggled me for quite some time. Because when added this way, the box-sizing setting is not respected, it's instead added on top of the 100% height, making the element higher than the viewport. On top of this the animations triggered correctly, but to me it looked like they triggered to early.

To sum it up. The data-trigger will fire correctly, when it  actually should. However, visually, the padding makes the element too high, so I couldn't see that it triggered at the right point.

This is for Skrollr 0.5.14, and might have been fixed later.

Not sure if this is a issue or just some bad practise on my part ;P I solved the problem by putting a sorounding div inside the first div, and the first still had height 100%, but only the second div got the padding-changes (no height set).
